### ISSUE TITLE
bug(auto-approve): crashed with `Not Found`

### ISSUE THREAD

**Issue by tmatsuo** [googleapis/repo-automation-bots#1783]
```
    HttpError: Not Found
        at /workspace/node_modules/@octokit/request/dist-node/index.js:66:23
        at async sendRequestWithRetries (/workspace/node_modules/@octokit/auth-app/dist-node/index.js:445:12)
        at async Object.getChangedFiles (/workspace/build/src/get-PR-info.js:29:17)
        at async /workspace/build/src/auto-approve.js:112:25
        at async Promise.all (index 0)
        at async /workspace/node_modules/gcf-utils/build/src/gcf-utils.js:229:21 
```

**Comment by tmatsuo** [comment]
I think it's an edge case where the PR head branch is gone, mostly due to force updating the branch.
We can just ignore 404.

**Comment by tmatsuo** [comment]
No blunderbuss?

**Comment by tmatsuo** [comment]
Well, it does nothing

**Comment by sofisl** [comment]
This was resolved in #2292, logs look clean now!
